### PR TITLE
UI enhancements

### LINKED ISSUE #23: My supported tokens: Add link to proposal
Each staked proposal info should link to the corresponding proposal detail page

### PR DESCRIPTION
**Pull request by fabriziovigevani**
closes #23 

- Updates colors which were using old accent color.
- Updated footer logo.
- Added link to corresponding proposal from `My supported proposals` component

**preview**

Screen Shot 2020-07-03 at 5 45 41 PM

Screen Shot 2020-07-03 at 5 41 10 PM

Screen Shot 2020-07-03 at 1 38 00 PM

### CODE REVIEW

File: src/assets/honey.svg [hunk 1]
```diff
@@ -1 +1 @@
...
...518 0 00-.76-1.304z" fill="#FFDD0F"/><path d="M21.8 44.944l-10.288-5.89a3.943 3.943 0 00-3.953.012l-10.25 5.962a3.967 3.967 0 00-1.967 3.425l.04 11.86a3.973 3.973 0 001.993 3.42L7.659 69.62a3.948 3.948 0 003.956-.011l10.252-5.963a3.978 3.978 0 001.957-3.431l-.04-11.859a3.974 3.974 0 00-1.985-3.413z" fill="#FFF19F"/><path d="M1.286-1.487l-6.359-3.64a2.438 2.438 0 00-2.444.007l-6.334 3.685A2.452 2.452 0 00-15.067.682l.025 7.33a2.454 2.454 0 001.231 2.114l6.356 3.64a2.44 2.44 0 002.445-.008l6.337-3.685a2.46 2.46 0 001.21-2.12L2.511.622a2.457 2.457 0 00-1.226-2.11z" fill="#FFDD0F"/><path d="M17.05-8.034l-6.357-3.641a2.438 2.438 0 00-2.444.008L1.915-7.983A2.451 2.451 0 00.699-5.865l.025 7.329a2.455 2.455 0 001.23 2.114l6.357 3.64a2.439 2.439 0 002.445-.007l6.337-3.685a2.46 2.46 0 001.21-2.12l-.026-7.33a2.456 2.456 0 00-1.226-2.11zm0 56.31l-6.357-3.64a2.438 2.438 0 00-2.444.008l-6.334 3.684a2.451 2.451 0 00-1.216 2.118l.025 7.329a2.455 2.455 0 001.23 2.114l6.357 3.64a2.44 2.44 0 002.445-.007l6.337-3.685a2.46 2.46 0 001.21-2.121l-.026-7.33a2.456 2.456 0 00-1.226-2.11z" fill="#FFEB6F"/></g><defs><clipPath id="clip0"><rect width="52" height="52" rx="26" fill="#fff"/></clipPath></defs></svg>
```
Comment (rperez89) [inline src/assets/honey.svg:1]: i think that we are sending again the same two svg? or we optimized them?

File: src/assets/logo.svg [hunk 2]
```diff
@@ -1 +1 @@
...
...8h-4.003zM145.363 39.294c0-1.523.232-2.855.695-3.998a8.583 8.583 0 011.83-2.856 7.589 7.589 0 012.606-1.73 8.091 8.091 0 013.026-.587c2.417 0 4.327.745 5.731 2.236 1.403 1.49 2.104 3.682 2.104 6.575 0 .283-.011.593-.032.93-.022.336-.043.636-.065.897h-10.943c.043.477.19.937.43 1.35.241.412.569.765.962 1.033.82.587 1.92.881 3.302.881.874.001 1.747-.08 2.606-.245a9.744 9.744 0 002.087-.604l.648 3.95a6.64 6.64 0 01-1.036.391c-.474.14-.955.255-1.441.343-.529.098-1.095.18-1.7.245-.602.065-1.207.098-1.812.098-1.533 0-2.866-.228-3.999-.685a7.78 7.78 0 01-2.816-1.877 7.444 7.444 0 01-1.651-2.823 11.261 11.261 0 01-.532-3.524zm11.331-1.86a5.054 5.054 0 00-.211-1.208 3.02 3.02 0 00-.55-1.044 2.947 2.947 0 00-.939-.751 3.04 3.04 0 00-1.409-.294 3.22 3.22 0 00-1.392.277 2.896 2.896 0 00-.971.735 3.3 3.3 0 00-.599 1.06 6.962 6.962 0 00-.307 1.225h6.378zM90.157 24.405c-.341.318-.7.618-1.072.899-.594.446-1.215.87-1.861 1.273-.647.402-1.31.772-1.991 1.11-.68.337-1.333.636-1.96.897l1.458 3.753a27.967 27.967 0 002.33-.93c.755-.335 1.48-.734 2.17-1.192v17.532h4.822V23.77h-.012l-3.884.635z" fill="#2C3437"/></g><defs><clipPath id="clip0"><path fill="#fff" d="M0 0h161.356v56H0z"/></clipPath></defs></svg>
```
Comment (rperez89) [inline src/assets/logo.svg:1]: same with this

File: src/ethereum-providers/icons/wallet.svg [hunk 3]
```diff
@@ -1 +1 @@
...
\ No newline at end of file
+<svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" fill="none"><path fill="#212B36" fill-rule="evenodd" d="M4 8.577c0-.329.266-.595.595-.595h14.81c.328 0 .595.266.595.595v10.815a.595.595 0 01-.595.595H4.595A.595.595 0 014 19.392V8.577zm1.19.595v9.625h13.62V9.172H5.19z" clip-rule="evenodd"/><path fill="#212B36" fill-rule="evenodd" d="M13.033 13.862a2.562 2.562 0 012.562-2.562h3.564c.329 0 .595.266.595.595v3.933a.595.595 0 01-.595.595h-3.564a2.562 2.562 0 01-2.562-2.561zm2.562-1.372a1.371 1.371 0 100 2.743h2.969V12.49h-2.97z" clip-rule="evenodd"/><path fill="#212B36" fill-rule="evenodd" d="M15.821 14.65a.778.778 0 100-1.556.778.778 0 000 1.557zm-.359-8.265c.078-.017.167-.04.25-.06l.05-.013a6.67 6.67 0 01.322-.073c.224-.045.413-.06.559-.04.131.02.196.062.242.123.058.076.146.263.146.686a.595.595 0 001.19 0c0-.568-.115-1.044-.384-1.402a1.514 1.514 0 00-1.024-.585c-.345-.05-.689-.004-.96.05a7.8 7.8 0 00-.438.1 6.18 6.18 0 01-.237.057L4.528 7.953a.595.595 0 00.295 1.153l10.639-2.721z" clip-rule="evenodd"/></svg>
```
Comment (rperez89) [inline src/ethereum-providers/icons/wallet.svg:1]: same here

File: src/assets/honey.svg (hunk 1 above)
Comment (fabriziovigevani) [reply src/assets/honey.svg:1]: Ah no, we just optimized them. Was my fault for not optimizing only the one added.

**Pull request merged**